assets/css/success.rtl.css

```diff
@@ -148,7 +148,7 @@
 	color: #4d3716;
 	justify-self: start;
 	width: max-content;
-	margin-left: 5px;
+	margin-right: 5px;
```

Comment:
Was this change intentional? It isn't mentioned in the PR description

Reply:
Yes, the command to build the project updated this file